backend/__tests__/vendorDeleteProperty.test.js

```
@@ -0,0 +1,189 @@
+const { deleteProperty } = require('../controllers/vendorControllers');
+const { User } = require('../models/userSchemas');
+const { Property } = require('../models/propertySchemas');
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)